chalice/deploy/deployer.py

```diff
@@ -141,15 +141,29 @@ def confirm(self, text, default=False, abort=False):
         # type: (str, bool, bool) -> bool
         return default
 
+    def Abort(self):
```

Comment:
Should I wrap `print`/`echo` up in there as well? Or is that outside the scope of a `prompter`?